components/brave_new_tab_ui/components/default/braveNews/customize/Modal.tsx

```diff
@@ -38,6 +38,12 @@ export default function BraveNewsModal () {
     } else if (!shouldRender && dialogRef.current?.open) {
       dialogRef.current?.close?.()
     }
+
+    const handleCancel = () => setCustomizePage(null)
+    dialogRef.current?.addEventListener('cancel', handleCancel)
```

Comment:
Note: Unfortunately we don't have up to date type definitions for the modal element, so typescript will complain if we just add an `onCancel` prop to the element :cry: